src/main/java/org/jalasoft/moi/controller/config/SwaggerConfig.java

```diff
@@ -11,16 +11,21 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+
 import springfox.documentation.builders.PathSelectors;
 import springfox.documentation.builders.RequestHandlerSelectors;
 import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.Contact;
+import springfox.documentation.service.VendorExtension;
 import springfox.documentation.spi.DocumentationType;
 import springfox.documentation.spring.web.plugins.Docket;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
- * This class implement basic configurations to get Swagger running in our project
- * and use it for specifications and tools that help us document our APIs.
+ * Implements basic configurations and documentations on Swagger user interface.
  *
  * @author Carlos Meneses
  * @version 1.1
```

Comment:
I think the version of this demo should be `1.2`

Reply:
Spring was used since demo 2. I think 1.1 is ok.